src/controllers/cron/update_local_smartapi.js

```diff
@@ -269,25 +283,28 @@ module.exports = () => {
     }
 
     if (!disable_smartapi_sync) {
-        cron.schedule('*/10 * * * *', async () => {
-            debug(`Updating local copy of SmartAPI specs now at ${new Date().toUTCString()}!`);
-            try {
-                await updateSmartAPISpecs();
-                debug("Successfully updated the local copy of SmartAPI specs.")
-            } catch (err) {
-                debug(`Updating local copy of SmartAPI specs failed! The error message is ${err.toString()}`)
-            }
-        });
+        const overridesPath = path.resolve(__dirname, "../../config/smartapi_overrides.json");
+        let overrides
+        try {
+            overrides = JSON.parse(fs.readFileSync(overridesPath));
+        } catch (error) {
+            debug(`ERROR getting API Overrides file because ${error}`);
+            return;
+        }
+
+        if(!(api_override && overrides.conf.only_overrides)) {
```

Comment:
This would appear to schedule the cron job only if overrides are not set. The cron job should be scheduled so long as `disable_smartapi_sync` is `false`